packages/graphql-transformer-common/src/FunctionResourceIDs.ts

```diff
@@ -1,13 +1,31 @@
 import { simplifyName } from './util';
 import md5 from 'md5';
+import { DirectiveNode } from 'graphql';
+import { getDirectiveArguments, TransformerContractError } from 'graphql-transformer-core';
+
+export interface FunctionDirectiveConfig {
```

Comment:
Can you please move this into a new file?

Reply:
Done.